facebook/react-native
# fix(ActionSheetIOS): allows using showActionSheetWithOptions in some app extensions

This fixes https://github.com/facebook/react-native/issues/29840. It ended up being much easier than I initially thought. I'm a bit confused as to why the action sheet does not use the RCTAlertController though.

## Summary

Allows using the ActionSheetIOS showActionSheetWithOptions inside an app extension granted the developer set up the override of the view controller.

## Changelog

[iOS] [Fixed] - ActionSheetIOS.showActionSheetWithOptions method works with RCTUtilsUIOverride.setPresentedViewController

## Test Plan

Setup an app extension in react-native. Then,
1. Use the action sheet without setting the override, see the updated message;
2. Use the action sheet with the override, it works! 🎉
